client/trino-client/src/main/java/io/trino/client/OkHttpUtil.java

```diff
@@ -108,6 +108,17 @@ public static Interceptor interceptRequest(Consumer<Request> consumer)
         };
     }
 
+    public static Interceptor httpsForwarded()
```

Comment:
This is not used by production code, so it should be moved to test code (under `src/test/java`). Maybe as utility method in `TestingTrinoClient`?

Reply:
Initially, I made this as an independent class because it can be used for even other than `TestingTrinoClient`. How about creating `HttpsForwardedInterceptor` in the testing library?

Reply:
@kokosing Rather, I moved this to the test case because this interceptor is not a big deal. We can refer to the test case as an example of how to test with forwarded https.

In that sense, the current commit message 'Allow TestingTrinoClient to use forwarded https' might not be suitable. The original 'Make OkHttpClient instance customizable at TestingTrinoClient' might be better, I feel.

Reply:
I think that current commit message express the goal of this commit, while the original was expressing the implementation.